shared/js/dataDriver.js

```diff
@@ -85,6 +85,18 @@ loop.DataDriver = function() {
     }
 
     /**
+      * Add a page to the room's table of contents.
+      *
+      * @param {String} url       The url of the page.
+      * @param {String} title     The title of the page.
+      * @param {Object} metadata  Additional metadata,
+      *                           as provided by PageMetadata.
+      */
+    addPageToRoom(url, title, metadata) {
```

Comment:
The "ToRoom" is making me wonder if we need to have "ToRoom" for all the other APIs. Or if it's unnecessary as all of these are adding data to the room?